avm/res/compute/virtual-machine/extension/main.json

```diff
@@ -5,8 +5,8 @@
   "metadata": {
     "_generator": {
       "name": "bicep",
-      "version": "0.31.34.60546",
-      "templateHash": "1194243367873711347"
+      "version": "0.31.92.45157",
```

Comment:
This only will not qualify for publishing

Reply:
@eriqua can't follow. What is needed in your opinion? The whole update is only, because Bicep released a new version. Nothing changed in the module.

Reply:
If no version to be released is stuck then it's fine, but it's important we're all aligned on how publishing works.

- If main.json is updated then a new version is published.
- If the pipeline fails, the version is not published until another PR is open, where the main.json is updated again.

So my comment here is, since avm/res/compute/virtual-machine/main.json is not updated in this PR, but it was in the previous one, which failed, the corresponding new version will not be published with this PR.

This is similar to the comment I added to #3832, but in that case is more relevant.

For this PR, if we want a new version to be published, we need a small update to avm/res/compute/virtual-machine/main.json as well. If that is not needed, then we're good to go and you can resolve this comment.

Reply:
Wow - I prepared a beautiful comment to +1 on @eriqua's point and instead of clicking on comment, I accidently selected 'resolve conversation'. Shame on me.

Also a shame that my text is gone - so let me start over in a short version:
- Your next action should be determined based on whether should should (have) lead to a new version being published or not
- If the PR was just added to fix an issue with the template with new intention of publishing a new version, there was no added action needed.
- If you actually wanted a new version (e.g., because previous changes were not published due to this template issue), then a dummy change (outside the module's metadata) would've been required. For example in the a module parameter's description.

@eriqua, I'm terribly sorry. @rahalan, please create or not create a new PR based on whether you wanted to have a new published version 😉